package.json

```diff
@@ -7,7 +7,9 @@
     "test": "test"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Awesome work man!

Can we add solium as a devDependency, as you mentioned in the MR description, please? And then I'm very happy to have this merged in :heart: